ParabolInc/action
# Fix stripe after rejected payment

fixes root cause of a null stripeSubscriptionId

TEST
- [ ] create a new org, upgrade to pro, it works
- [ ] change billing info (test cards here: https://stripe.com/docs/testing). it works


This is tedious, sorry!

TEST - FAILED PAYMENT
- run `yarn start:tunnel` requires ultrahook, check your .env & install via http://www.ultrahook.com/
- in stripeFailPayment, comment out L25 (throw Error...)
- in StripeManager, uncomment L24 (trial_end...)
- Create a new org, upgrade them with CC `4000 0000 0000 0341` with any valid expiry & cvv
- after 15 seconds, visit http://localhost:3000/me/organizations/:orgId & verify that the invoice is pending. if not pending, keep refreshing
- click "See Details" for the pending invoice, copy the invoiceId at the end of the url (looks like `in_1EgKSjFLaFINmHnBFAYC1kK4`)

- go to /admin/graphql, run the following mutation (change the invoiceId to yours):
```
mutation {
  stripeFailPayment(invoiceId: "in_1EgKSjFLaFINmHnBFAYC1kK4") {
    error {
      message
    }
  }
}
```
- [ ] refresh the org page, see that the invoice is marked as failed
- [ ] goto the team dash, see that there is modal preventing usage  
- [ ] update the CC number (`4242424242424242`)
- [ ] refresh the org billing page after 15 seconds, see 3 invoices: a failed one, a pending one, and an upcoming one.
